docs/design/schema.md

```diff
@@ -0,0 +1,50 @@
+# Schema
+
+UOR Schema is the attribute type declaration for a UOR Collection. Schema also links application logic to a importing UOR Collection.
```

Comment:
Perhaps mention specific uses here.WDYT?. Example: "The schema is a validation tool for UOR collections"